x-pack/plugins/endpoint/public/applications/endpoint/store/endpoint_list/reducer.ts

```diff
@@ -5,32 +5,51 @@
  */
 
 import { Reducer } from 'redux';
-import { EndpointListState } from './types';
+import { ManagementState } from '../../types';
 import { AppAction } from '../action';
 
-const initialState = (): EndpointListState => {
+const initialState = (): ManagementState => {
   return {
     endpoints: [],
-    request_page_size: 10,
-    request_index: 0,
+    pageSize: 10,
+    pageIndex: 0,
     total: 0,
+    loading: false,
   };
 };
 
-export const endpointListReducer: Reducer<EndpointListState, AppAction> = (
+export const endpointListReducer: Reducer<ManagementState, AppAction> = (
```

Comment:
we should rename this for consistency

Reply:
@parkiino we're going with `Management` naming, aren't we?  @peluja1012 what would we change here?

Reply:
@kevinlog I think the exported `const` is what needs to be renamed.

Reply:
you are correct - ignore me